config/sync/dblog.settings.yml

```diff
@@ -1,3 +1,3 @@
-row_limit: 10000
+row_limit: 1000
```

Comment:
Is the reduction here intentional? I reckon we can spare an extra 9000 db rows with all the disk space we have available :)

Reply:
Well, it was intentional, my thinking was that we were now being more selective about what we kept for the long term, so we didn't need to use such a 'shotgun approach' !
But maybe you're right, I'll change it back so that we can have our cake and eat it !